<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-19T09:42:45 IST" name="Default suite" finished-at="2022-12-19T09:45:31 IST" duration-ms="165926">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2022-12-19T09:42:45 IST" name="Default test" finished-at="2022-12-19T09:45:31 IST" duration-ms="165926">
      <class name="cucumberOptions.TestNGTestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" started-at="2022-12-19T09:42:45 IST" name="setUpClass" finished-at="2022-12-19T09:42:46 IST" duration-ms="1589" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3ec9f8d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" started-at="2022-12-19T09:42:46 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-12-19T09:43:47 IST" duration-ms="60550" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Verification of the account details page for both the new accounts created"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Account details verification"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ParaBank | Error] but found [ParaBank | Accounts Overview]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ParaBank | Error] but found [ParaBank | Accounts Overview]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at pageObjects.AccountsPage.verifyLogin(AccountsPage.java:30)
at stepDefinitions.AccountPageStepDefinition.user_verfy_login_is_successful(AccountPageStepDefinition.java:23)
at ✽.user verify login is successful(file:///D:/ParaBank/ParaBank/src/test/java/features/accountVerification_B.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" started-at="2022-12-19T09:43:47 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-12-19T09:44:38 IST" duration-ms="51313" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Verification of opening two new accounts for both account types - CHECKING and SAVINGS."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Opening account"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ParaBank | Error] but found [ParaBank | Accounts Overview]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ParaBank | Error] but found [ParaBank | Accounts Overview]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at pageObjects.AccountsPage.verifyLogin(AccountsPage.java:30)
at stepDefinitions.AccountPageStepDefinition.user_verfy_login_is_successful(AccountPageStepDefinition.java:23)
at ✽.user verify login is successful(file:///D:/ParaBank/ParaBank/src/test/java/features/openingAccount_A.feature:8)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" started-at="2022-12-19T09:44:38 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-12-19T09:45:31 IST" duration-ms="52227" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Verify that the Balance and transaction table details are correct in each of the accounts"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Bill Pay"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ParaBank | Error] but found [ParaBank | Accounts Overview]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ParaBank | Error] but found [ParaBank | Accounts Overview]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at pageObjects.AccountsPage.verifyLogin(AccountsPage.java:30)
at stepDefinitions.AccountPageStepDefinition.user_verfy_login_is_successful(AccountPageStepDefinition.java:23)
at ✽.user verify login is successful(file:///D:/ParaBank/ParaBank/src/test/java/features/transactionVerification_C.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:cucumberOptions.TestNGTestRunner@909217e]" started-at="2022-12-19T09:45:31 IST" name="tearDownClass" finished-at="2022-12-19T09:45:31 IST" duration-ms="144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- cucumberOptions.TestNGTestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
